pandas/core/series.py

```diff
@@ -547,6 +547,66 @@ def __len__(self):
         return len(self._data)
 
     def view(self, dtype=None):
+        """
+        Create a new view of the Series.
+
+        This function will return a new Series with a view of the same
+        underlying values in memory, optionally reinterpreted with a new data
+        type. The new data type must preserve the same size in bytes as to not
+        cause index misalignment.
+
+        Series are instantiated with `dtype=float64` by default, so unlike
+        `numpy.array.view()` this function will not try to preserve the current
+        Series data type.
```

Comment:
This part I don't fully understand. Can you try to explain more what you exactly want to say, maybe with an example? (here in the comments)

Reply:
It's basically what I said in the first comment of this PR. `pd.Series([1,2,3], dtype='int8').view()` will raise, but the equivalent in numpy would not.

Reply:
Ah, yes, thanks for the explanation. In principle, doing `view` without specifying a dtype is not that useful. So I would keep this explanation for the notes or so, and not include it here in the extended summary.

Reply:
Notes looks like a good fit. On it.